serving/samples/grpc-ping-go/README.md

```diff
@@ -2,32 +2,27 @@
 
 A simple gRPC server written in Go that you can use for testing.
 
-This sample is dependent on
```

Comment:
Change this to say that this only works in Knative 0.4.0?

Reply:
Done